templates/job-application-email.php

```diff
@@ -16,4 +16,4 @@
 }
 ?>
 <?php // translators: %1$s is the email address, %2$s is the subject query args. ?>
-<p><?php printf( wp_kses_post( __( 'To apply for this job <strong>email your details to</strong> <a class="job_application_email" href="mailto:%1$s%2$s">%1$s</a>', 'wp-job-manager' ) ), esc_html( $apply->email ), '?subject=' . rawurlencode( $apply->subject ) ); ?></p>
+<p><?php printf( wp_kses_post( __( 'To apply for this job <strong>email your details to</strong> <a class="job_application_email" href="mailto:%1$s%2$s">%1$s</a>', 'wp-job-manager' ) ), esc_html( $apply->email ), '?subject=' . rawurlencode( htmlspecialchars_decode( $apply->subject ) ) ); ?></p>
```

Comment:
This is a very small detail, but..could you please change `htmlspecialchars_decode` to [`wp_specialchars_decode`](https://developer.wordpress.org/reference/functions/wp_specialchars_decode/)? I feel it should be more appropriate for the task, especially as it's a WordPress function that does basically the same task.

Here's my suggestion:

```suggestion
<p><?php printf( wp_kses_post( __( 'To apply for this job <strong>email your details to</strong> <a class="job_application_email" href="mailto:%1$s%2$s">%1$s</a>', 'wp-job-manager' ) ), esc_html( $apply->email ), '?subject=' . rawurlencode( wp_specialchars_decode( $apply->subject, ENT_QUOTES  ) ) ); ?></p>
```